examples/redux-loop/redux-loop-example/package.json

```diff
@@ -3,23 +3,29 @@
   "version": "2.5.0",
   "private": true,
   "devDependencies": {
-    "react-scripts": "^1.0.14"
+    "npm-link-shared": "^0.5.3",
+    "react-scripts": "^2.0.5",
+    "string": "^3.3.3"
   },
   "dependencies": {
     "immutable": "^3.8.2",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-redux": "^5.0.6",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-redux": "^5.1.0",
     "react-redux-subspace": "^2.5.0",
-    "redux": "^3.7.2",
-    "redux-act": "^1.5.0",
-    "redux-loop": "^3.2.2",
+    "redux": "^4.0.1",
+    "redux-act": "^1.7.4",
+    "redux-loop": "^4.4.1",
     "redux-subspace-loop": "^2.5.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
-    "test": "echo 'No Tests'"
-  }
+    "prestart": "npm-link-shared ./node_modules/redux-subspace-loop/node_modules . redux-loop",
```

Comment:
at some point during redux-loop v3, they started using a`Symbol` for marking if a reducer returned a loop.  The issue with that for us is that `Symbol` created in one file is not the same reference as an identical `Symbol` created in another and because lerna symlinks `redux-subspace-loop` into the example, it has it's own copy of redux-loop so the `Symbol`s are no longer the same so loops started to no look like loops anymore.  This hack just symlinks the version or `redux-loop` from `redux-subspace-loop` into the example so they are the same file again.